addons/repair/tests/test_repair.py

```diff
@@ -186,16 +181,16 @@ def test_01_repair_states_transition(self):
         with self.assertRaises(UserError):
             repair.action_validate()
 
-        #  Line A with qty > 0 & not available, Line B with qty >= 0 & available --> Warning (stock.warn.insufficient.qty.repair)
-        lineA.product_uom_qty = 2.0
-        lineB = self._create_simple_part_move(repair.id, 2.0, self.product_storable_lot)
+        lineB = self._create_simple_part_move(repair.id, 1.0, self.product_storable_no2)
         repair.move_ids |= lineB
-        quant = self.create_quant(self.product_storable_no, 1)
-        quant |= self.create_quant(self.product_storable_lot, 3)
-        quant.action_apply_inventory()
 
-        lineC = self._create_simple_part_move(repair.id, 1.0, self.product_storable_order_repair)
```

Comment:
```suggestion
        lineC = self._create_simple_part_move(repair.id, 1.0, self.product_storable_no2)
```
You can only change this line and drop every other change in this test, as it's confusing/unecessary to invert two variables for the fun of it. :confused:

Reply:
You're right; I reverted it and it still works. I think I fiddled with it too much last time, trying to make sense of it all. Thanks!